compiler/one-cmds/onecc.template.cfg

```diff
@@ -2,6 +2,10 @@
 [Environment]
 ONECC_ENV="ONECC"
 
+[backends]
```

Comment:
below example uses `[backend]` but this line has `s`

Reply:
Oh, my bad. I'll change it to `backend`.